--- origsrc/ktoblzcheck-1.18/src/python/ktoblzcheck.py	2008-03-11 04:28:51.000000000 -0500
+++ src/ktoblzcheck-1.18/src/python/ktoblzcheck.py	2008-06-17 23:09:59.406250000 -0500
@@ -33,7 +33,7 @@
 - IbanCheck.bic_position() raises a ValueError when the IBAN is unuseable.
 """
 
-import os
+import os, sys
 from ctypes import c_void_p, c_char_p, c_int, c_ulong, cdll, POINTER, byref
 #from enum import Enum
 
@@ -42,7 +42,10 @@
 try:
     kto = cdll.ktoblzcheck
 except OSError:
-    kto = cdll['libktoblzcheck.so.1']
+    if sys.platform == 'cygwin':
+        kto = cdll['cygktoblzcheck-1.dll']
+    else:
+        kto = cdll['libktoblzcheck.so.1']
 
 
 class Record(c_void_p):
